src/com/sindhu/webapplication/ToDoList.java

```diff
@@ -24,43 +24,50 @@
         }
         return toDoListIds;
     }
+
     public static List<ToDoItem> toDoListItems(String username) {
         List<ToDoItem> toDoList = new ArrayList<>();
         try {
-            String query = "SELECT ID, toDoItem FROM To_Do_Lists WHERE username=" + "'" + username + "'";
+            String query = "SELECT ID, toDoItem, timer FROM To_Do_Lists WHERE username=" + "'" + username + "'";
             Connection connection = Database.getConnection();
-            if(connection == null){
+            if (connection == null) {
                 return toDoList;
             }
             Statement statement = connection.createStatement();
             ResultSet resultSet = statement.executeQuery(query);
             while (resultSet.next()) {
-                toDoList.add(new ToDoItem(resultSet.getLong(1), resultSet.getString(2)));
+                resultSet.getString("timer");
+                resultSet.wasNull();
+                if (resultSet.wasNull()) {
+                    toDoList.add(new ToDoItem(resultSet.getLong("ID"), resultSet.getString("toDoItem"), ""));
+                } else {
+                    toDoList.add(new ToDoItem(resultSet.getLong("ID"), resultSet.getString("toDoItem"), resultSet.getString("timer")));
+                }
             }
         } catch (SQLException sql) {
-            System.out.println("error");
             System.out.println("Error in sql statement :" + sql.getMessage());
         }
         return toDoList;
     }
-    public static  void addItem(String username,String addItem){
+
+    public static void addItem(String username, String addItem) {
         try {
             String insert = "INSERT INTO To_Do_Lists(username , toDoItem)" +
                     "VALUES (?, ?)";
             Connection connection = Database.getConnection();
-            if(connection == null){
+            if (connection == null) {
                 return;
             }
             PreparedStatement ps = connection.prepareStatement(insert);
             ps.setString(1, username);
             ps.setString(2, addItem);
             int rowsUpdated = ps.executeUpdate();
-            if(rowsUpdated == 1){
+            if (rowsUpdated == 1) {
```

Comment:
we are not closing the connection if `rowsUpdated == 1`. We should follow a standard here. 
Either close connection everytime you are done with a sql statement or return the connection to `DatabaseV2` so that it can reuse/close that connection. (not just here, in all other places too). 

1. call `connection.close`
2. or implement a function called `DatabaseV2.recycleConnection` and call that function. inside that function, for now, just call `connection.close`. in the future, we can implement recycling functionality.